docs/For Developers/Building NW.js.md

```diff
@@ -86,6 +86,8 @@ See the upstream documentation for the mapping between GN and GYP flags: https:/
 
 ## Generate ninja build files with GYP for Node
 
+Note: Required default Python version set to Python2.7.
```

Comment:
We should probably write Python `2.7.6` since that's what the [Buildbot](http://buildbot-master.nwjs.io:8010/about) states.

Reply:
Do you know the plan to update the python version to version 3.x? It is quite unfortunate that we have to revert our dev machine's python version in order to run the scripts from nwjs.

Reply:
No idea myself but Roger can weigh in on that.

Reply:
New command and python3 needed here in the latest version. Please check the official build log: http://buildbot-master.nwjs.io:8010/builders/nw67_sdk_win64/builds/7/steps/node_gyp/logs/stdio